js/src/analysis/isSEMrushIntegrationActive.js

```diff
@@ -0,0 +1,14 @@
+import getL10nObject from "./getL10nObject";
+
+import { get } from "lodash-es";
+
+/**
+ * Returns whether or not the SEMrush integration is active
+ *
+ * @returns {boolean} Whether or not the SEMrush integration is active.
+ */
+export default function isSEMrushIntegrationActive() {
+	const l10nObject = getL10nObject();
+
+	return get( l10nObject, "semrushIntegrationActive", 0 ) === 1;
```

Comment:
To my understanding, on the PHP side the default value is `true` while here the default is `0`? Please double check.

Reply:
I'm mimicking what happens in e.g. `js/src/analysis/isKeywordAnalysisActive.js` (which is enabled by default).
I suppose that the reason behind that is that if we don't get a value for the option something's wrong and it's better to disable the features...

Reply:
Thanks for the clarification! I now see it is part of a larger change in another PR https://github.com/Yoast/wordpress-seo/commit/5678750d69d9b56da62ce2052dc823d4e2fe0c08#